src/quote/composer.json

```diff
@@ -6,20 +6,20 @@
         "php": ">= 8.3",
         "ext-json": "*",
         "ext-pcntl": "*",
-        "monolog/monolog": "3.5.0",
+        "monolog/monolog": "3.8.1",
         "open-telemetry/api": "1.0.3",
```

Comment:
I wonder why dependabot is not bumping this dependency 🤔 

Latest is `1.2.0`: https://packagist.org/packages/open-telemetry/api